feature/feature.go

```diff
@@ -64,6 +64,10 @@ const (
 	// of clusters selected by a placement. For more info check ManifestWorkReplicaSet APIs
 	ManifestWorkReplicaSet featuregate.Feature = "ManifestWorkReplicaSet"
 
+	// ManifestWorkReplicaSetCloudEventsDrivers will enable the cloud events drivers (mqtt or grpc) for the
+	// ManifestWorkReplicaSet controller, so that the controller can send manifestworks through cloud events.
+	ManifestWorkReplicaSetCloudEventsDrivers featuregate.Feature = "ManifestWorkReplicaSetCloudEventsDrivers"
```

Comment:
might be just CloudEventsDrivers. addon manager might need to pick this featuregate also.

Reply:
updated name.